content/docs/user-guide/components/reference/atom/bloom.md

```diff
@@ -1,20 +1,38 @@
 ---
+linkTitle: Bloom
 title: Bloom Component
-linktitle: Bloom
-description: "Open 3D Engine (O3DE) Bloom component reference."
+description: The Bloom component, which simulates real-world light bleeding (glow), is provided by the Atom Gem in Open 3D Engine (O3DE). 
 toc: true
 ---
 
-The **Bloom component** simulates real-world light bleeding, or glow.
+The **Bloom** component creates *bloom*, a post-processing effect that simulates real-world light bleeding, or glow.
 
 
-## Provider ##
+## Provider
 
-[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
+[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom)
+
+
+## Dependencies
+
+[PostFX Layer component](./postfx-layer)
 
 
 ## Properties
 
-{{< todo issue=https://github.com/o3de/o3de.org/issues/717 >}}
-Add properties.
-{{< /todo >}}
\ No newline at end of file
+![Bloom component interface](/images/user-guide/components/reference/atom/bloom-component-ui.png)
+
+
+### Base properties
+
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Enable Bloom** | If enabled, activates the bloom effect. | Boolean | The default value |
+| **Threshold** | Bloom effect applies only to pixels with a brightness value greater than this threshold. | Float: 0.0 to Infinity  | `1.0` |
+| **Knee** | Creates a gradual transition between the pixels below and above the threshold. This softens and spreads the bloom effect.  | Float: 0.0 to 1.0 | `0.5` |
+| **Intensity** | Scales the intensity of the bloom effect. | Float: 0.0 to Infinity | `0.5` |
+| **Enable Bicubic** | If enabled, applies bicubic filtering. This can help reduce undesired artifacts that may appear. | Boolean |  False |
```

Comment:
```suggestion
| **Enable Bicubic** | If enabled, applies bicubic filtering. This can help reduce undesired artifacts that may appear. | Boolean |  `Disabled` |
```